### ISSUE TITLE
better-docs theme for the primary font and a few brand colors

### ISSUE THREAD

**Issue by headwinds** [SoftwareBrothers/better-docs#74]
I like the layout and appreciate all the thought that was gone into the design of this project. 

I see that its possible to customize the look and feel in the Readme but it doesn't mention passing in a theme config. Is a custom logo and then black & white my only option branding?

Similar to Material UI, have you considered passing a simple theme config to change primary fonts and colors?

**Comment by headwinds** [comment]
I'll provide a little more detail...

I've have a toy design system project where I'm trying to learn how to auto generate the docs. If I can get working for my toy project, then I'll attempt to convince my co-workers to adopt it. 

So I was able to add my own branding in this demo...

[demo](https://cross-country-docs.now.sh/)

...and here's my hack

After some research into jsdocs, I learned about the `layout.tmpl` file and I was able to go into the node_modules directory and find your version and then add my custom styles so that they show up on every page. Now, when I run `yarn docs` I get my desired styling. 

So this could close this issue but just curious if there's a better or you might consider adding this as a way to add your own branding. 

Your `category` tag is brilliant! It really works well with an atomic design approach.
